pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -178,10 +224,8 @@ func (rc *reconciler) reconcile() {
 						rc.maxWaitForUnmountDuration)
 				}
 			}
-			if err != nil &&
-				!nestedpendingoperations.IsAlreadyExists(err) &&
-				!exponentialbackoff.IsExponentialBackoff(err) {
-				// Ignore nestedpendingoperations.IsAlreadyExists && exponentialbackoff.IsExponentialBackoff errors, they are expected.
+			if err != nil && !exponentialbackoff.IsExponentialBackoff(err) {
+				// Ignore exponentialbackoff.IsExponentialBackoff errors, they are expected.
```

Comment:
The same thought was the reason to remove the check. Please note that the check was there to actually **prevent** the logging of the error, as it was probably considered "normal" that it happened.

Before this PR, the reconciler would try to start the operation every 100ms, only succeeding the first time and erroring for every try that followed. This check was the reason that there was never an error message shown.

Now that we have the `if rc.attacherDetacher.IsOperationPending(attachedVolume.VolumeName, "")` check a few lines above, this error should not happen anymore and if it does for what ever reason, it should be logged.